bloom-housing/bloom
# 3291/listing export take 2

# Pull Request Template

## Issue Overview

This PR addresses #3291 


## Description

This PR adds the listings export functionality along with associated tests! Column information here: https://docs.google.com/spreadsheets/d/1iMhybYs0RSdW940e402oI8xsqot8BlAYHaVTXaORIeA/edit#gid=670984196
 
## How Can This Be Tested/Reviewed?

This can be tested by clicking the export to csv button on the partners side and verifying that the listing data is displayed correctly. Additionally, create your own listing to test cases that may not be present in our seed.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
